HelpMyStreetFE/HelpMyStreetFE/src/js/shared/cookie-helper.js

```diff
@@ -40,7 +47,7 @@ function deleteUneccsaryCookies(setCookieAfterDelete) {
         var name = eqPos > -1 ? cookie.substr(0, eqPos) : cookie;
         var isNeccesary = (neccesaryCookies.find(x => name.trim().startsWith(x.trim())) != undefined);        
         if (cookie && !isNeccesary)
-            document.cookie = name + "=; path=/; domain=" + window.location.hostname + "; expires=Thu, 01 Jan 1970 00:00:00 GMT";
+            document.cookie = name + "=;expires=Thu, 01 Jan 1970 00:00:00 GMT";
```

Comment:
Did you mean to remove the domain?

Reply:
yeah, it wasnt working in Live because the domains are all slightly different, apparently the new line will remove any cookies belonging to the domain as long as their path is "/" and they are not "HttpOnly"